.travis.yml

```diff
@@ -1,47 +1,51 @@
 language: python
 sudo: false
-python:
-  - 2.7_with_system_site_packages
-  - 3.4
-addons:
-  apt:
-    packages:
-    - python-qt4
-    - python-qt4-dev
-    - python-qt4-gl
-    - python-pip
-    - python-numpy
-    - python-wxtools
-    - ccache
-    - cmake
-env:
-  - ETS_TOOLKIT=qt4
-  - ETS_TOOLKIT=null
-  - ETS_TOOLKIT=wx
 matrix:
+  include:
+    - python: 2.7
+      env: RUNTIME=2.7 TOOLKIT=null
+    - python: 2.7
+      env: RUNTIME=2.7 TOOLKIT=pyqt
+    - python: 2.7
+      env: RUNTIME=2.7 TOOLKIT=pyside
+    - python: 2.7
+      env: RUNTIME=2.7 TOOLKIT=wx
+    - python: 2.7
+      env: RUNTIME=3.5 TOOLKIT=null
+    - python: 2.7
+      env: RUNTIME=3.5 TOOLKIT=pyqt
+    - language: generic
+      os: osx
+      env: RUNTIME=2.7 TOOLKIT=null
+    - language: generic
+      os: osx
+      env: RUNTIME=2.7 TOOLKIT=pyqt
+    - language: generic
+      os: osx
+      env: RUNTIME=2.7 TOOLKIT=pyside
+    - language: generic
+      os: osx
+      env: RUNTIME=2.7 TOOLKIT=wx
+    - language: generic
+      os: osx
+      env: RUNTIME=3.5 TOOLKIT=null
+    - language: generic
+      os: osx
+      env: RUNTIME=3.5 TOOLKIT=pyqt
   allow_failures:
-    - env: ETS_TOOLKIT=wx
+    - env: RUNTIME=2.7 TOOLKIT=wx
+  fast_finish: true
 cache:
   directories:
-    - $HOME/.cache
-    - $HOME/.ccache
+    - $HOME/.edm
```

Comment:
This is not the default edm cache. With this we actually cache the edm environments which is not a good idea. Using the egg cache is fast enough